dont_have_vinogret.py

```diff
@@ -0,0 +1,28 @@
+
+import datetime
+import random
+
+
+def random_date_in_range_str(start, end):
+    """
+    The function choose random date in the range between the 2 given dates
+    :param start: start date - String!!!
+    :param end: end date - String!!!
+    :return: random date between the 2 dates
+    """
+    list_dates = []
+    list_dates = start.split("-")
+    start_date = datetime.date(int(list_dates[0]), int(list_dates[1]), int(list_dates[2]))
```

Comment:
https://www.educative.io/edpresso/how-to-convert-a-string-to-a-date-in-python 
refactor using built in logic of the module

Reply:
It didn't work when you use is YYYY-MM-DD and not the DD-MM-YYYY